### ISSUE TITLE
get_hist() is not correct

### ISSUE THREAD

**Issue by lackBfun** [toimcio/SegNet-tensorflow#11]
I think the get_hist() function is not correct. You should use the predicted label rather than logits as input.
get_hist() function is mainly used to calculate the number of true-positive, false-positive, true-negative, and false-negative samples. It should return a K*K (K is the number of classes) matrix where the x-axis is predicted classes while the y-axis is ground-truth classes.
